### PR TITLE
Video next up

### PR DESCRIPTION
**Pull request by i-like-robots**
screen shot 2016-09-07 at 12 03 28

Displays the next up panel when the video ends if there is another video in the playlist. The panel will display for 5 seconds before being removed and triggering the next video or clicking it will skip the wait. On iOS (or other devices that trigger the `playsFullScreen` flag) nothing will happen.

This PR also simplifies some of the video play logic for a more conistent experience across devices and better delivery of ads.

### CODE REVIEW

File: client/components/video-scroller/player.js [hunk 1]
```diff
@@ -77,6 +79,53 @@ function onVideoClick(e){
 	e.stopPropagation();
 }
 
+function onVideoEnd () {
+	if (state.playsFullScreen) {
+		return;
+	}
+
+	let index = state.playlist.currentIndex + 1;
```
Comment (ironsidevsquincy) [inline client/components/video-scroller/player.js]: `const`?

**Comment by leggsimon** [comment]
Since there's no closing of full screen vids. I can close https://github.com/Financial-Times/next-front-page/pull/1025 ?

File: client/components/video-scroller/player.js [hunk 2]
```diff
@@ -32,92 +34,101 @@ export function init(){
...
-	let placeholder = wrapper.querySelector('.o-video__placeholder');
-	video.style.width = placeholder.offsetWidth+'px';
-	video.style.height = placeholder.offsetHeight+'px';
-	wrapper.replaceChild(video, placeholder);
+function showUpNext (next) {
+	const container = state.elements.upNext = document.createElement('div');
+
+	container.className = 'video-scroller__next-up';
+
+	container.innerHTML = `
```
Comment (aintgoin2goa) [inline client/components/video-scroller/player.js]: This doesn't seem like a great idea - yeah, it's the easiest way to get it done but someone (possibly your future self) is gonna curse you for putting this here.

Could we at least move to the top of the file or something?

Comment (i-like-robots) [inline client/components/video-scroller/player.js]: Reckon it would be better to have this inactive in the DOM?

Comment (aintgoin2goa) [inline client/components/video-scroller/player.js]: Yes, that's much better I think

Comment (i-like-robots) [inline client/components/video-scroller/player.js]: Refactored this into the server-side templates

**Pull request merged**